CHANGELOG.md

```diff
@@ -1647,7 +1651,7 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 
 - **CUMULUS-670**
 
-  - The behavior of ParsePDR and related code has changed in this release. PDRs with FILE_TYPEs that do not conform to the PDR ICD (+ TGZ) (https://cdn.earthdata.nasa.gov/conduit/upload/6376/ESDS-RFC-030v1.0.pdf) will fail to parse.
+  - The behavior of ParsePDR and related code has changed in this release. PDRs with FILE_TYPEs that do not conform to the PDR ICD (+ TGZ) (<https://cdn.earthdata.nasa.gov/conduit/upload/6376/ESDS-RFC-030v1.0.pdf)> will fail to parse.
```

Comment:
Is (<url)> correct?

Reply:
(assuming so, coming back to it)

Reply:
Sorry, all of these ancillary markdown "fixes" were "auto" applied, and I didn't notice the URL fixes were not fixed correctly. I'll revert all of the changelog changes except for my entry describing the specific change for CUMULUS-1894.